sdk/eventhub/Azure.Messaging.EventHubs/src/EventHubClient.cs

```diff
@@ -28,6 +28,12 @@ public class EventHubClient : IAsyncDisposable
         /// <summary>The policy to use for determining retry behavior for when an operation fails.</summary>
         private EventHubRetryPolicy _retryPolicy;
 
+        /// <summary>
+        ///   The name of the host used to connect to the associated Event Hubs namespace.
+        /// </summary>
+        ///
+        public string EventHubsHostName { get; }
```

Comment:
Maybe we should think about something like `FullyQualifiedHostName` or similar here.  Ramya had run that by the service team and they seemed to really like it.   

That should also refactor the constructor argument name too.

@ramya-rao-a:  Can you confirm that I got that name right?

Reply:
I believe it is `FullyQualifiedNamespace`.

Reply:
That sounds familiar.    Lets go with that and rename the existing parameter on Event Hub client to match, if you'd be so kind

Reply:
Yes, `FullyQualifiedNamespace` is what we landed on this morning